### DISCUSSION TITLE
how can I handle error with useQueries?

### DISCUSSION THREAD

**Discussion by LydiaYoon** [TanStack/query#4549] [Q&A]
I did it like this using `useQuery`
```JavaScript
useQuery(
  [queryKey1],
  queryFn1,
  {
    suspense: true,
    useErrorBoundary: (error) => error.response?.status >= 500,
  }
)
useQuery(
  [queryKey2],
  queryFn2,
  {
    suspense: true,
    useErrorBoundary: true,
  }
)
```
`status < 500` doesn't need `Error Boundary`.
However, I need parallel processing within suspense to prevent network waterfall, so have to use `useQueries`.

```JavaScript
useQueries({
  queries: [
    {
      queryKey: queryKey1,
      queryFn: queryFn1,
      useErrorBoundary: (error) => error.response?.status >= 500, // error
    },
    {
      queryKey: queryKey2,
      queryFn: queryFn2,
      useErrorBoundary: true,
    },
  ],
})
```
but, `useErrorBoundary` property doesn't get Error parameter
In this case, how can I handle error with `useQueries`?

**Comment by TkDodo** [comment]
> but, useErrorBoundary property doesn't get Error parameter

it should. can you show a codesandbox reproduction?
